README.md

```diff
@@ -1,81 +1,262 @@
+<li>When fully grown, a plant is ready for harvest by a farmer. Farmers deposit plants in the nearest silo (grey-blue cylinder).</li>
+<li>When enough resources have been collected into a silo, a new farmer spawns from the silo.</li>
+<li>Farmers look for their next task in a limited radius. If no task is found, they look in an increasingly larger radius until a task is found.</li>
+<li>For every fifth farmer spawned, a drone is spawned instead.</li>
+<li>Drones fly and harvest plants. They do not destroy rocks or plant crops.</li>
+<li>After depositing, drones hover over the silo until they locate a plant to harvest.</li>
+<li>Keyboard controls allow the user to reset the simulation.</li>
+   </ul>
+</details>
+ 
+![Auto Farmers](_imgs/AutoFarmers.png?raw=true)
+ 
+ <details>
+   <summary><strong>Bucket Brigade</strong>: Firefighters pass buckets along a chain to extinguish a fire.
+   </summary>
+     
+   <ul>
+<li>Each cell of the grid has a temperature from 0.0 to 1.0. Cells above 0.2 are on fire and transmit heat to their neighbors. The color and height of a fire cell reflects its temperature. (For visual effect, a fire cell's height randomly oscilates a small amount, but this does not reflect its actual temperature.)</li>
+<li>On start, a few random cells are on fire. Buckets are randomly placed around the field.</li>
+<li>Workers are grouped into teams. Each team picks the closest cell on fire and forms two lines from the fire to the nearest water (blue cells outside the field). One line passes full buckets; the other passes empty buckets.</li>
+<li>Each team has a bucket fetcher, who retrieves the nearest bucket, walks it to the water at the end of the line, fills the bucket, and then drops it.</li>
+<li>Workers in the 'pass full' line look for full buckets on the ground within a small radius. When found, they walk over, pick up the bucket, carry it to the next worker up the line, and drop it. The last worker in the 'pass full' line empties the bucket on the fire (lowering the temperature of that cell and its eight surrounding cells to zero). Workers move slower while carrying a full bucket.</li>
+<li>Workers in the 'pass empty' line behave the same but only carry empty buckets. The last worker in the 'pass empty' line will fill the bucket in the nearest water.</li>
+<li>Every frame, a team will reposition its line to the fire cell closest to its current water source.</li>
+<li>When a team's water source is exhausted, it repositions the line to the next closest water.</li>
+<li>When repositioning, a worker will not resume its normal behavior until it has reached its new position.</li>
+<li>In the moment between a worker placing a bucket on the ground and the next picking it up, the line might reposition, in which case the bucket will be left on the ground.</li>
+<li>OPTIONAL: A configured small number of omniworkers pick up the nearest bucket, fill it at the nearest water source, and dump it on the nearest fire.</li>
```

Comment:
I don't understand the term "omniworker", does that mean independent worker (without a team)?